multisig/proposals/zksyncmainnet/vip-004/index.ts

```diff
@@ -0,0 +1,340 @@
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMarketSupplyCaps(address[],uint256[])", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMarketBorrowCaps(address[],uint256[])", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLiquidationIncentive(uint256)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setCloseFactor(uint256)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setMinLiquidatableCollateral(uint256)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setActionsPaused(address[],uint256[],bool)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "addPool(string,address,uint256,uint256,uint256)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setPoolName(address,string)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "updatePoolMetadata(address,VenusPoolMetaData)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setProtocolSeizeShare(uint256)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setReserveFactor(uint256)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setInterestRateModel(address)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "updateJumpRateModel(uint256,uint256,uint256,uint256)", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setRewardTokenSpeeds(address[],uint256[],uint256[])", zksyncmainnet.GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "setLastRewardingBlock(address[],uint32[],uint32[])", zksyncmainnet.GUARDIAN],
+    },
```

Comment:
The signature `setLastRewardingBlock(address[],uint32[],uint32[])` is not checked anymore.

`setLastRewardingBlocks(address[],uint32[],uint32[])` and `setLastRewardingBlockTimestamps(address[],uint256[],uint256[])` should be used instead

Reply:
[05c0740](https://github.com/VenusProtocol/vips/pull/377/commits/05c07404f0bca9365e9700754f8e089bde142f76)